### PR TITLE
Add pgweb to Easypanel 🎉 

### PR DESCRIPTION
**Pull request by Supernova3339**

### CODE REVIEW

File: templates/pgweb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: pgweb
+description: Cross-platform client for PostgreSQL databases
+instructions: null
+changeLog:
+  - date: 2023-3-15
+    description: first release
+links:
```
Comment (Ponkhy) [inline templates/pgweb/meta.yaml:7]: You could also add the Demo page `https://pgweb-demo.fly.dev`.

**Comment by Ponkhy** [comment]
Maybe add some fields for the `DATABASE_URL` env var, so people can fix their database connection?

**Comment by Supernova3339** [comment]
That's not needed at all, it's for autoconnection

File: templates/pgweb/meta.yaml (hunk 1 above)
Comment (Supernova3339) [reply templates/pgweb/meta.yaml:7]: not implemented

**Review by Ponkhy — APPROVED** [review]
LGTM

**Pull request merged**